tests/users.js

```diff
@@ -18,6 +18,14 @@ beforeEach(() => knex.migrate.rollback()
 				token = res.body.token;
 			})
 	)
+	.then(() =>
+		request(app)
```

Comment:
This chunk can go in the same section as the previous one, since we don't need to wait for the first token request to finish before we fire off the second one.  Maybe just generalize this to get an array of tokens?